src/spec/payid-protocol.md

```diff
@@ -270,15 +270,15 @@ This specification defines the PayID protocol - an application-layer protocol, w
   In addition to common Headers, the PayID client MUST specify the following request header.
 
 ### Header Accept
-  The PayID client MUST specify the `Accept` request header field with at least one of the registered media types (Section X). The purpose of this header is to indicate what type of content can be understood in the response. It specifies the `payment-network` and `environment` of the payment account and its representation format for which the PayID client wants to receive information. The representation format is always JSON.
+  The PayID client MUST specify the `Accept` request header field with at least one of the registered media types (Section X). The purpose of this header is to indicate what type of content can be understood in the response. It specifies the `paymentNetwork` and `environment` of the payment account and its representation format for which the PayID client wants to receive information. The representation format is always JSON.
 
   The PayID server MUST reject formats that specify unknown or unsupported format parameters.
 
 ## Response Headers
   In addition to the Common Headers, the PayID server MUST specify the following response header.
 
 ### Header Cache-Control
-  The PayID server MUST include the `Cache-Control` header with the max-age limit of 0. The intermediate hops or PayID client MUST NOT cache the responses.
+  The PayID server MUST include the `Cache-Control` header with a value `Cache-Control: no-store`. PayID responses MUST NOT be stored in any cache.
```

Comment:
`Cache-Control: no-store` is the canonical way to say that something should not be stored in any cache. This is the response the PayID server sends, and what the 3rd party https://payidvalidator.com checks for.